### DISCUSSION TITLE
SEVERE and WARNING logs after fresh install

### DISCUSSION THREAD

**Discussion by talcher** [Keyfactor/ejbca-ce#383] [Q&A]
Hello Keyfactor,

I have freshly installed EJBCA-CE-8.0, with WildFly-26.0 on a Debian 11 V.M. I followed the [official documentation](https://doc.primekey.com/ejbca/ejbca-installation/).

However, whenever I (re)start the application server, I always get 2 SEVERE logs and 75 WARNING logs.

The SEVERE logs are:
> SEVERE [javax.enterprise.resource.webcontainer.jsf.flow] (MSC service thread 1-1) Unable to obtain CDI 1.1 utilities for Mojarra
> SEVERE [javax.enterprise.resource.webcontainer.jsf.application.view] (MSC service thread 1-1) Unable to obtain CDI 1.1 utilities for Mojarra

1 WARNING log is:
> WARN  [org.jboss.as.jaxrs] (MSC service thread 1-4) WFLYRS0018: Explicit usage of Jackson annotation in a Jakarta RESTful Web Services deployment; the system will disable Jakarta JSON Binding processing for the current deployment. Consider setting the 'resteasy.preferJacksonOverJsonB' property to 'false' to restore Jakarta JSON Binding.

12 WARNING logs look like this one:
> WARN  [io.undertow.servlet] (ServerService Thread Pool -- 57) UT015020: Path /* is secured for some HTTP methods, however it is not secured for [HEAD, POST, GET]

The remaining WARNING logs look like this one:
> WARN  [org.jboss.as.ejb3] (MSC service thread 1-2) WFLYEJB0525: The 'mappedName' in Jakarta Enterprise Beans  annotations is not supported. Value of 'ejbca/RoleMemberSessionRemote' for Jakarta Enterprise Beans 'RoleMemberSessionBean' will be ignored.

My questions are:
- are those logs expected, or have I made mistakes while setting up EJBCA ?
- are there any security impacts related to those logs? Especially the SEVERE ones?

Thank you for your answer.

talcher

**Comment by mike-agrenius-kushner** [comment]
Hi, 

No, all of the above is mostly some startup babble from Wildfly and components, as well as some deprecated annotations we haven't removed yet. They can be safely ignored. 

Cheers,
Mike

**Comment by talcher** [comment]
Those are great news.
Thank you, Mike, for your quick answer.
